src/test/regress/upgrade/upgrade_test.py

```diff
@@ -0,0 +1,139 @@
+"""upgrade_test
+Usage:
+    upgrade_test --old-bindir=<old-bindir> --new-bindir=<new-bindir> --postgres-srcdir=<postgres-srcdir> [--citus-path=<citus-path>]
+
+Options:
+    --old-bindir=<old-bindir>              The old PostgreSQL executable directory;(ex: '~/.pgenv/pgsql/bin')
+    --new-bindir=<new-bindir>              New postgres binary absolute path(ex: '~/.pgenv/pgsql-11.3/bin')
+    --postgres-srcdir=<postgres-srcdir>    Path to postgres build directory(ex: ~/.pgenv/src/postgresql-11.3/src/test/regress)
+    --citus-path=<citus-path>              Absolute path for citus directory (default: ~/citus)
```

Comment:
What do we use the `--citus-path` for? Quick search doesn't show any usage

Reply:
It is a leftover, I will remove it.